### ISSUE TITLE
Stack render broken?

### ISSUE THREAD

**Issue by sarahkadar** [meteor-factory/react-native-tinder-swipe-cards#84]
I am not sure what I am doing wrong.

Without stack, looking good:
![2017-04-18 00 34 12](https://cloud.githubusercontent.com/assets/48946/25107403/746b2d30-23cf-11e7-8df9-73e4d29fb839.png)

With stack, all messed up:
![2017-04-18 00 36 32](https://cloud.githubusercontent.com/assets/48946/25107410/79171498-23cf-11e7-8f16-3ce4efb73df8.png)

Code is at https://github.com/KTamas/imdbinder/blob/master/Tinder.js

**Comment by esganzerla** [comment]
I'm not really sure why the `renderStack` function had these values inside, so I'm not making a PR, but I found that if you change the lines 331-335 (SwipeCards.js) to:

```javascript
      let offsetX = "50%"
      let lastOffsetX = this.props.stackOffsetX;

      let offsetY = "50%"
      let lastOffsetY = this.props.stackOffsetY;
```

it was originally:
```javascript
      let offsetX = this.props.stackOffsetX * cards.length - i * this.props.stackOffsetX;
      let lastOffsetX = this.props.stackOffsetX;

      let offsetY = this.props.stackOffsetY * cards.length - i * this.props.stackOffsetY;
      let lastOffsetY = this.props.stackOffsetY;
```

I believe it will work just fine. still scaling as well

**Comment by sarahkadar** [comment]
@esganzerla thanks, this fixes it! sort of. don't look very stack-ey anymore.

Though now I have yep/nope shown somehow reversed... and maybe is simply not working...

**Comment by pbassut** [comment]
@esganzerla That's exactly what gives the stacky look of it. Every card available gets slightly shifted  to the lower-right.

**Comment by sarahkadar** [comment]
I figured out what I messed up: some default styles of create-react-native-app mess with it. so completely my fault.

**Comment by pbassut** [comment]
@KTamas mind sharing what that was?

**Comment by sarahkadar** [comment]
the default [CRNA](https://github.com/react-community/create-react-native-app) `App.js` has this:

```
const styles = StyleSheet.create({
  container: {
    flex: 1,
    backgroundColor: '#fff',
    alignItems: 'center',
    justifyContent: 'center',
  },
});
```

`alignItems: center` and `justifyContent: center` messes up `react-native-tinder-swipe-cards`.

**Comment by pbassut** [comment]
Right. I'm not using CRNA tho.

**Comment by sarahkadar** [comment]
Yeah. The default app.js add this style to the main container and that breaks things so it's not really a CRNA but more like a stylesheet issue I guess.

**Comment by YannzizTay** [comment]
@KTamas you need to set the stackOffsetX and you will be good  try out stackOffsetX={2}
